src/exposedComponents/types.ts

```diff
@@ -19,6 +19,7 @@ export interface OpenInExploreTracesButtonProps {
 export interface EmbeddedTraceExplorationState extends SharedExplorationState {
   initialTimeRange: TimeRange;
   onTimeRangeChange?: (timeRange: TimeRange) => void;
+  urlSync?: boolean;
```

Comment:
Where are we toggling this on - another PR?

Reply:
Yep - it's passed in https://github.com/grafana/asserts-app-plugin/pull/1411

Reply:
Thanks, wasn't 100% sure if that was the final PR equivalent to this one :)